hash/msx2_cart.xml

```diff
@@ -880,10 +948,12 @@ LZ93A13 (32 pin) - 8KB banks
 	</software>
 
 	<software name="mississp">
-		<description>Mississippi Satsujin Jiken (Japan)</description>
+		<description>Murder on the Mississippi (Japan)</description>
```

Comment:
Did the game actually use this English title officially?  If not, it’s better to use transliterated title.

Reply:
It did on the box at least: https://www.generation-msx.nl/software/activision/murder-on-the-mississippi/release/1023/